clang/include/clang/Basic/Attr.td

```diff
@@ -380,6 +380,13 @@ class Clang<string name, bit allowInC = 1, int version = 1>
   bit AllowInC = allowInC;
 }
 
+// This spelling combines the spellings of GCC and Clang for cases where the
+// spellings are equivalent for compile compatibility.
+class ClangGCC<string name, bit allowInC = 1, int version = 1>
```

Comment:
Nice! Could have probably used this for `NoStackProtector`, too.

`NoSanitizeSpecific` was almost a candidate, but the attribute has different spellings between compiler namespaces. Same for `AcquireCapability`.